tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2019_10_29/native_with_function_name.pysnap

```diff
@@ -65,4 +65,3 @@ system:
           "EXC_BAD_ACCESS / KERN_INVALID_ADDRESS"
         value (ignored because stacktrace takes precedence)
           "Fatal Error: EXC_BAD_ACCESS / KERN_INVALID_ADDRESS"
-      threads (thread has no stacktrace)
```

Comment:
What makes this line disappear? That looks fishy.

Reply:
certain "static" non-contributing components are only generated for the app variant. We have verified this together in code.